### ISSUE TITLE
Linear atelectasis

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#6696]
**Preferred term label:**
Linear atelectasis

**Synonyms**
Discoid atelectasis
Platelike atelectasis

**Definition (free text, please give PubMed ID)**

Linear atelectasis is a focal area of subsegmental atelectasis with a linear configuration, almost always extending to the pleura
(74). It is commonly horizontal but sometimes oblique or vertical. The thickness of the atelectasis may range from a few millimeters to more than 1 cm. Linear atelectasis is also referred to as (See also atelectasis.)
PMID:18195376

Comment
See Figure 33 of Hansell DM, et al., Fleischner Society: glossary of terms for thoracic imaging. Radiology. 2008 Mar;246(3):697-722.

Parent term (use hpo.jax.org/app)
Abnormal pulmonary thoracic imaging finding HP:0031983
?? Atelectasis alone?

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
